app.go

```diff
@@ -72,9 +76,20 @@ func main() {
 		Desc:   "The topic to write the concept annotation to",
 		EnvVar: "PRODUCER_TOPIC",
 	})
+	whitelistRegex := app.String(cli.StringOpt{
+		Name:   "whitelistRegex",
+		Desc:   "The regex to use to filter messages based on Origin-System-Id.",
+		EnvVar: "WHITELIST_REGEX",
+		Value:  "http://cmdb.ft.com/systems/methode-web-pub",
```

Comment:
Strictly, if this is a regex, periods and other special characters should be escaped, because they should match literally here.